src/asciidocParser.ts

```diff
@@ -91,10 +91,12 @@ export class AsciidocParser {
 
   public convertUsingJavascript (
     text: string,
-    doc: vscode.TextDocument,
-    context: vscode.ExtensionContext,
-    editor: vscode.WebviewPanel
-  ): {html: string, document: Asciidoctor.Document} {
+    doc: SkinnyTextDocument,
+    forHTMLSave: vscode.ExtensionContext,
+    backend: vscode.WebviewPanel,
+    context?: vscode.ExtensionContext,
+    editor?: vscode.WebviewPanel
```

Comment:
This is incorrect, probably due to the rebase.
It should be:

```suggestion
    context: vscode.ExtensionContext,
    editor: vscode.WebviewPanel
```

Reply:
it took me a while but I get it now :sweat_smile: